src/schema-templates/preparation-time.block.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Recipe preparation time block schema template.
+ *
+ * @package Yoast\WP\SEO\Schema_Templates
+ */
+
+use Yoast\WP\SEO\Schema_Templates\Assets\Icons;
+// phpcs:disable WordPress.Security.EscapeOutput -- Reason: The Icons contains safe svg.
+?>
+{{block name="yoast/preparation-time" title="<?php esc_attr_e( 'Preparation time', 'wordpress-seo' ); ?>" category="yoast-structured-data-blocks" parent=[ "yoast/recipe" ] description="<?php esc_html_e( 'The recipe\'s preparation time', 'wordpress-seo' ); ?>" icon="<?php echo Icons::heroicons_clock(); ?>" supports={"multiple": false} }}
```

Comment:
You should use FQN global function calls in namespaced files (i.e., use `\` before WordPress and PHP functions). Please check all occurrences.

(There is a sniff for this, but it's from an external standard which has a PHP 7.1 minimum PHP version, which is why we can't add it to YoastCS, as it would conflict with the minimum PHP version of the plugins.)